content/id/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Melakukan Reservasi Sumber Daya Komputasi untuk Daemon Sistem
+content_type: task
+min-kubernetes-server-version: 1.8
+---
+
+<!-- overview -->
+
+Node Kubernetes dapat dijadwalkan sesuai dengan kapasitas. Secara bawaan, Pod dapat menggunakan
+semua kapasitas yang tersedia pada sebuah Node. Ini merupakan masalah karena Node
+sebenarnya menjalankan beberapa _daemon_ sistem yang diperlukan oleh OS dan Kubernetes itu sendiri.
+Jika sumber daya pada Node tidak direservasi untuk _daemon-daemon_ tersebut, maka
+Pod dan _daemon_ akan berlomba-lomba menggunakan sumber daya yang tersedia, sehingga
+menyebabkan _starvation_ sumber daya pada Node.
```

Comment:
translating starvation to "kekurangan" will make it loss context and meaning, because "starvation" is not just about lacking resource. I think in general operating system topic people use "starvation" and "deadlock" without translation, wdyt?

example: http://ftp.gunadarma.ac.id/linux/docs/v06/Kuliah/SistemOperasi/BUKU/SistemOperasi-4.X-1/ch17s07.html